pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -1007,10 +1063,11 @@ def bpm(self, filename, det, shape=None, msbias=None):
         # Call the base-class method to generate the empty bpm
         bpm_img = super().bpm(filename, det, shape=shape, msbias=msbias)
 
-        if det == 1:
-            # TODO: This is for the 1x1 binning it should
-            # change for other binning
-            bpm_img[:2369,1326:1328] = 1.
+        # TODO -- come back to this
```

Comment:
Is this to be addressed in this PR, or another?

Reply:
A future one.  I don't see any significant things to mask so far.
  Have modified the comment.